doc/content/xenopsd/walkthroughs/VM.start.md

```diff
@@ -225,8 +225,8 @@ module and looks for scripts in the hardcoded path `/etc/xapi.d`.
 ## 2. create a Xen domain
 
 The `VM_create` micro-op calls the `VM.create` function in the backend.
-In the classic Xenopsd backend the
-[VM.create_exn](https://github.com/xapi-project/xenopsd/blob/b33bab13080cea91e2fd59d5088622cd68152339/xc/xenops_server_xen.ml#L633)
+In the classic Xenopsd backend, the
+[VM.create_exn](https://github.com/xapi-project/xen-api/blob/master/ocaml/xenopsd/xc/xenops_server_xen.ml#L1421-L1586)
```

Comment:
It might be worth linking to a particular commit hash here (and below) so that it'd be easier to track down what lines are being talked about in the future.

Reply:
done.